themes/default/content/docs/guides/continuous-delivery/gitlab-app.md

```diff
@@ -23,6 +23,11 @@ To enable the integration with your GitLab project, you will need to ensure you
   in Pulumi.
   * After you add the organization, ensure that it uses GitLab [as its identity provider]({{< relref "/docs/intro/pulumi-service/organizations#changing-identity-providers" >}}).
 
+{{% notes type="warning" %}}
+If you have [Merged Results Pipeline](https://docs.gitlab.com/ee/ci/pipelines/pipelines_for_merged_results.html) enabled for your project you must disable
+it. For more information, see the [issue](https://gitlab.com/gitlab-org/gitlab/-/issues/350086) on GitLab.
```

Comment:
i
```suggestion
it. For more information, see the [GitLab issue](https://gitlab.com/gitlab-org/gitlab/-/issues/350086) on GitLab.
```

do we need a tracking issue to remove this once the issue is resolved?

Reply:
> do we need a tracking issue to remove this once the issue is resolved?

Yeah I think so